src/Resources/public/sonata-page.back.js

```diff
@@ -1,10 +1,25 @@
-/*
- * This file is part of the Sonata Project package.
+/**
+ *
+ * This file is part of the Sonata package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * generated on: Fri Dec 01 2017 00:48:06 GMT+0100 (CET)
+ * revision:     4e8bf4373af18c69b99332d84f3cff6894faa869
+ *
+ */
+/**
```

Comment:
can you please remove the duplicate doc block?

Reply:
This file is auto generated :/

Reply:
by which command?

Reply:
Just go to the `assets_src` directory and call `yarn install` and `gulp`

Reply:
hmm, here it is working fine in your PR: `src/Resources/public/sonata-page.front.css`